### PR TITLE
Add handling of projects limits

### PR DESCRIPTION
**Pull request by artsiomtserashkovich**
## Before submitting checklist

- [x] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [x] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: src/neptune/internal/backends/swagger_client_wrapper.py [hunk 1]
```diff
@@ -80,6 +80,9 @@ def handle_neptune_http_errors(response, exception: Optional[HTTPError] = None):
             "WORKSPACE_IN_READ_ONLY_MODE": lambda response_body: NeptuneLimitExceedException(
```
Comment (Raalsky) [inline src/neptune/internal/backends/swagger_client_wrapper.py:80]: Let's differentiate the workspace-level and project-level exceptions

Comment (normandy7) [reply src/neptune/internal/backends/swagger_client_wrapper.py:80]: I could also update the current `NeptuneLimitExceedException` message to be more helpful for each scenario, unless there's some technical reason for the split?

Comment (artsiomtserashkovich) [reply src/neptune/internal/backends/swagger_client_wrapper.py:80]: I would say it's not related to technical, it's more about whether is it OK (from a user perspective) to use one exception for both cases.

Comment (Raalsky) [reply src/neptune/internal/backends/swagger_client_wrapper.py:80]: Yes. It's fine to have only `NeptuneLimitExceedException` as long as it covers both situations :wink:

Comment (normandy7) [reply src/neptune/internal/backends/swagger_client_wrapper.py:80]: Alright, I think that would be fine. I'll add a proposal tomorrow.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## [UNRELEASED] neptune 1.6.3
+
+### Changes
+- Add handling of projects limits [#1456](https://github.com/neptune-ai/neptune-client/pull/1456)
```
Comment (normandy7) [inline CHANGELOG.md:4]:
```suggestion
- Add handling of project limits [#1456](https://github.com/neptune-ai/neptune-client/pull/1456)
```

File: src/neptune/exceptions.py [hunk 3]
```diff
@@ -823,15 +823,23 @@ def __init__(self, reason: str):
...
-    {bash}neptune sync -p project_name{end}
-What should I do?
-    - In case of storage limitations, go to your projects and remove runs or model metadata you don't need
-    - ... or update your subscription plan here: https://app.neptune.ai/-/subscription
-You may also want to check the following docs page:
+Data synchronization was interrupted because you're out of storage space. If you're using Neptune in asynchronous mode
+(default), the data that couldn't be uploaded is safely stored on the disk. You can still fetch and delete data from
+your project(s).
+How can I free up space in Neptune?
+    - Contact your workspace admin about possible project or workspace storage limits.
```
Comment (normandy7) [inline src/neptune/exceptions.py:830]: For the record: IMO we can live with this in the client until the feature is released in full.

**Review by normandy7 — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
+## [UNRELEASED] neptune 1.6.3
```
Comment (Raalsky) [inline CHANGELOG.md:1]: Looks like someone already released that. Please, remove the `UNRELEASED` and move it to the `1.6.4` section.

**Review by Raalsky — APPROVED** [review]

**Pull request merged**